src/tree/help/HelpsTreeItem.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { AzExtParentTreeItem, AzExtTreeItem, GenericTreeItem, IActionContext } from "vscode-azureextensionui";
+import { localize } from "../../localize";
+import { OpenUrlTreeItem } from "../OpenUrlTreeItem";
+
+export class HelpsTreeItem extends AzExtParentTreeItem {
+    public label: string = 'help';
+    public contextValue: string = 'help';
+
+    public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
+        return [
+            // this.getStartedTreeItem,
+            this.readDocumentationTreeItem,
+            this.reviewIssuesTreeItem,
+            this.reportIssuesTreeItem
+        ];
+    }
+
+    public hasMoreChildrenImpl(): boolean {
+        return false;
+    }
+
+    public compareChildrenImpl(item1: AzExtTreeItem, item2: AzExtTreeItem): number {
+        // default sorting is based on the label which is being displayed to user.
+        // use id to control the order being dispalyed
+        return item1.id.localeCompare(item2.id);
+    }
+
+    // private get getStartedTreeItem(): AzExtTreeItem {
+    //     const node = new OpenUrlTreeItem(
+    //         this,
+    //         localize('views.help.getStarted', 'Get Started'),
+    //         'https://aka.ms/helppanel_getstarted',
+    //         new vscode.ThemeIcon('star-empty'));
+    //     node.id = '1';
+
+    //     return node;
+    // }
```

Comment:
Should there be a comment here to explain when this code should be uncommented?

Reply:
Added a comment.